src/api/route-services/experiment.js

```diff
@@ -46,6 +46,30 @@ class ExperimentService {
     return data;
   }
 
+  async getListOfExperiments(experimentIds) {
+    const dynamodb = createDynamoDbInstance();
+
+    const params = {
+      RequestItems: {
+        [this.experimentsTableName]: {
+          Keys: [...experimentIds].map((experimentId) => convertToDynamoDbRecord({ experimentId })),
+        },
+      },
+    };
+
+    try {
+      const response = await dynamodb.batchGetItem(params).promise();
```

Comment:
The `batchGetItem`s have some restrictions that we have to deal with, for the previous use case of `batchGetItem` we created a ticket. We should add this use case to that ticket too.
https://biomage.atlassian.net/browse/BIOMAGE-992